src/main/java/controller/LottoController.java

```diff
@@ -0,0 +1,51 @@
+package controller;
+
+import domain.Cash;
+import domain.LottoGame;
+import domain.LottoGameResult;
+import domain.LottoNumber;
+import domain.LottoTicket;
+import domain.LottoTickets;
+import domain.WinningLottoTicket;
+import ui.LottoView;
+
+public class LottoController {
+
+	public void play() {
+		LottoTickets lottoTickets = buyLottoTickets();
+
+		WinningLottoTicket lottoWinningNumbers = inputLottoWinningTicket();
+
+		LottoGameResult lottoGameResult = runLottoGame(lottoTickets, lottoWinningNumbers);
+
+		displayLottoGameResult(lottoGameResult);
+	}
+
+	private static void displayLottoGameResult(LottoGameResult lottoGameResult) {
+		LottoView.displayResult(lottoGameResult);
+		LottoView.displayYield(lottoGameResult.calculateYield());
+	}
+
+	private static LottoGameResult runLottoGame(LottoTickets lottoTickets,
+		WinningLottoTicket lottoWinningNumbers) {
+		LottoGame lottoGame = new LottoGame(lottoTickets, lottoWinningNumbers);
+		LottoGameResult lottoGameResult = lottoGame.calculateResult();
+		return lottoGameResult;
```

Comment:
아래와 같이 변수 선언없이 바로 return 하도록 변경해볼 수 있을 것 같아요. 🙂
```suggestion
		return lottoGame.calculateResult();
```


IntelliJ(=IDE)가 이런 부분들을 표시해주는데 이런 부분을 주의깊게 확인해보면 도움이 되실거에요. 😃

<img width="896" alt="image" src="https://github.com/next-step/java-lotto-kakao/assets/47850258/4a9d1405-805c-4e16-b582-627b61ba5f1c">


<img width="842" alt="image" src="https://github.com/next-step/java-lotto-kakao/assets/47850258/d87f6e77-d270-4b22-a7b2-e5071c2ef84a">